sql/pre_install/tables.sql

```diff
@@ -126,6 +126,7 @@ CREATE TABLE _timescaledb_catalog.dimension_slice (
   -- table constraints
   CONSTRAINT dimension_slice_pkey PRIMARY KEY (id),
   CONSTRAINT dimension_slice_dimension_id_range_start_range_end_key UNIQUE (dimension_id, range_start, range_end),
```

Comment:
Arguably, this unique constraint could be dropped, its index though is still useful, and the `UNIQUE` isn't wrong, so we should just let it be?

Reply:
the unique is at least misleading so we should drop it, but we need to keep the index as that is used internally